iris-mpc-cpu/bin/hnsw_algorithm_metrics.rs

```diff
@@ -61,7 +63,13 @@ async fn main() -> Result<(), Box<dyn Error>> {
         .register_dynamic(param_openings, Operation::OpenNode)
         .init();
 
-    tracing_subscriber::registry().with(counting_layer).init();
+    let file = File::create("forest.txt")?;
+    let file_processor = PrettyPrinter::new().writer(std::sync::Mutex::new(file));
+
+    tracing_subscriber::registry()
+        .with(counting_layer)
+        .with(ForestLayer::new(file_processor, NoTag {}).with_filter(EnvFilter::new("cpu_time")))
```

Comment:
Maybe the target name could be the default (the whole `searcher` module)? Otherwise an HNSW-specific name.

Reply:
Good point. Switched to `searcher::cpu_time` as I want to add network-related metrics for the same module in a later PR.